.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	trailingComma: 'es5', // 배열, 오브젝트 프로퍼티 등에서 마지막 항목에도 콤마를 붙일지를 설정
+	tabWidth: 4, // tab 1 = space 4
+	useTabs: true, // tab과 space 중 무엇을 사용할 것인가
+	semi: true, // semicolon(;) 자동완성
+	singleQuote: true, // '' 와 "" => '' 로 통일
+	arrowParens: 'avoid', // 화살표함수 인자가 하나일 때 parenthesis() 생략
+};
```

Comment:
```suggestion
module.exports = {
	trailingComma: 'all',
	tabWidth: 2,
	useTabs: false,
	semi: true, // semicolon(;) 자동완성
	singleQuote: true, // '' 와 "" => '' 로 통일
	arrowParens: 'always',
};
```
[airbnb 규칙](https://github.com/airbnb/javascript)에 따르면 포맷을 이렇게 해야할 것 같습니다. 아래는 참고 문서입니다!

* [airbnb whitespace](https://github.com/airbnb/javascript#whitespace)
* [airbnb trailing commas](https://github.com/airbnb/javascript#commas--dangling)
* [airbnb arrow parens](https://github.com/airbnb/javascript#arrows--one-arg-parens)

Reply:
첨부까지 해주시다니... 감사합니다!!! 👍